src/components/DatePicker/RangePicker.jsx

```diff
@@ -168,6 +237,15 @@ export const RangePicker = ({
               <>
                 {moment.utc(value.start).format(displayFormat)} &divide;{' '}
                 {moment.utc(value.end).format(displayFormat)}
+                {multiple && isComparing && (
+                  <>
+                    {' '}
+                    {labelCompareDivider}{' '}
+                    {moment.utc(compareValue.start).format(displayFormat)}{' '}
+                    &divide;{' '}
+                    {moment.utc(compareValue.end).format(displayFormat)}
```

Comment:
I would put the `{' '}` on new lines so that it's more readable?

Reply:
I tries .... but prettier prefers it this way :)